ui/components/multichain/import-token-link/__snapshots__/import-token-link.test.js.snap

```diff
@@ -1,5 +1,42 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Import Token Link should match snapshot for goerli caipChainId 1`] = `
```

Comment:
need to check this against the old snapshot since it thinks the snapshot is new because i renamed the scenario